email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -709,7 +710,22 @@ ipcRenderer.on('update-available', () => {
   emitter.emit(Event.UPDATE_AVAILABLE, { value: true });
 });
 
-/* Window events: handle
+/* Window events: handle */
+ipcRenderer.on('check-network-status', () => {
+  const isOnline = window.navigator.onLine;
```

Comment:
Isn't there a way to do this in the main process without needing an ipc call to the renderer?

Reply:
`window.navigator.*` is only available for browser environments